src/librustc/hir/def.rs

```diff
@@ -142,7 +142,6 @@ pub enum Res<Id = hir::HirId> {
     Upvar(Id,           // `HirId` of closed over local
           usize,        // index in the `freevars` list of the closure
           ast::NodeId), // expr node that creates the closure
```

Comment:
Btw, I've started a PR removing `Res::Upvar` just in case you wanted to do that too.